### PR TITLE
Add Python linting for modules

### PR DESCRIPTION
**Pull request by redhatrises**

### CODE REVIEW

File: plugins/module_utils/falconctl_utils.py [hunk 1]
```diff
@@ -50,7 +50,7 @@ def __get(opt):
     opt = opt.replace("_", "-")
     cmd.append("--%s" % opt)
     try:
-        stdout = check_output(cmd, universal_newlines=True, stderr=STDOUT)
+        stdout = check_output(cmd, universal_newlines=True, stderr=STDOUT, shell=False)
```
Comment (redhatrises) [inline plugins/module_utils/falconctl_utils.py:53]: @carlosmmatos will you check that this change doesn't affect things?

File: plugins/modules/falconctl.py [hunk 2]
```diff
@@ -177,31 +179,33 @@ def __init__(self, module):
...
         return value
 
-
     def __run_command(self, cmd):
         rc, stdout, stderr = self.module.run_command(
             cmd, use_unsafe_shell=False)
 
+        if stderr:
+            self.module.fail_json(rc=rc, stderr=stderr)
+
```
Comment (redhatrises) [inline plugins/modules/falconctl.py:197]: @carlosmmatos same here?

File: plugins/module_utils/falconctl_utils.py (hunk 1 above)
Comment (carlosmmatos) [reply plugins/module_utils/falconctl_utils.py:53]: This is fine

File: plugins/modules/falconctl.py [hunk 3]
```diff
@@ -313,20 +316,20 @@ def validate_params(self, params):
...
 
 
-def main():
+def main():  # pylint: disable=missing-function-docstring
     module_args = dict(
         state=dict(required=True, choices=[
                    "absent", "present"], type="str"),
-        cid=dict(required=False, no_log=False, type="str"),
-        provisioning_token=dict(required=False, type="str"),
+        cid=dict(required=False, no_log=True, type="str"),
```
Comment (carlosmmatos) [inline plugins/modules/falconctl.py:323]: do we consider CID to be sensitive output in a play?

**Comment by carlosmmatos** [comment]
LGTM

**Pull request merged**